docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM continuumio/anaconda3:4.1.1
+MAINTAINER "Greg Way" <gregway@mail.med.upenn.edu>
+
+# Install base packages and R
+RUN apt-get update --fix-missing && apt-get install -y \
```

Comment:
Looks like [MRO + conda](http://conda.pydata.org/docs/mro.html) could be a simple way of dealing with this. I will need look into this more and implement a solution - see #16